src/example/index.js

```diff
@@ -15,9 +15,9 @@ import VSHADER_SOURCE_LINES from './vshader_lines.glsl';
 import FSHADER_SOURCE_LINES from './fshader_lines.glsl';
 import CheckerBoard from './check64.png';
 
-// let theModel = getModelData(new THREE.SphereGeometry(1, 8, 8));
-let theModel = getModelData(new THREE.CubeGeometry());
-// let theModel = getModelData(new THREE.PlaneGeometry());
+ let theModel = getModelData(new THREE.SphereGeometry(1, 8, 8));
+// let theModel = getModelData(new THREE.CubeGeometry(1, 1, 1, 1, 1, 1));
```

Comment:
gives the cube model more data. Last three parameters allow more vertices per face.